packages/app/src/app/pages/Patron/PricingModal/PricingChoice/ThankYou/index.tsx

```diff
@@ -18,13 +18,9 @@ export const ThankYou: React.FC<ThankYouProps> = ({
   <Margin bottom={2}>
     {!markedAsCancelled && <Title color={color}>Awesome!</Title>}
     <SubTitle>
-      {markedAsCancelled ? (
-        'Your subscription will be automatically cancelled before your next billing date.'
-      ) : (
-        <>
-          Thank you <strong>so</strong> much for your support of ${price}!
-        </>
-      )}
+      {markedAsCancelled
+        ? 'Your subscription will be automatically cancelled before your next billing date.'
+        : `Thank you <strong>so</strong> much for your support of ${price}!`}
```

Comment:
This will render `strong` has well. maybe something like this?

```suggestion
        : <span>Thank you <strong>so</strong> much for your support of {price}!</span>}
```